dotnet/tools/common/manifest.c

```diff
@@ -193,8 +193,9 @@ void CreateLinkIfNeeded(const char *target, const char *toCreate)
 
 void LinkFiles(const char *manifestDir)
 {
+    manifestDir;
```

Comment:
For C++ code I would just comment out the parameter name. But that does not work for C. The compiler complains "expected formal parameter list, not a type list"